tests/foreman/destructive/test_contenthost.py

```diff
@@ -46,8 +46,8 @@ def test_content_access_after_stopped_foreman(target_sat, rhel7_contenthost):
             target_sat.cli_factory.YumRepository(url=settings.repos.yum_1.url),
         ],
     )
-    repos_collection.setup_content(org.id, lce.id, upload_manifest=False, override=True)
-    repos_collection.setup_virtual_machine(rhel7_contenthost, install_katello_agent=False)
```

Comment:
How is `install_katello_agent` related to the context of the PR? :thinking:

Reply:
It's not, but that argument is removed from this line in the other branches since katello-agent was removed in 6.15. Maybe some wire got crossed when I cherry picked to 6.14? Not sure. Regardless, I will add that back in.